src/pudl/analysis/record_linkage/model_helpers.py

```diff
@@ -1,18 +1,134 @@
 """This module provides shared tooling that can be used by all record linkage models."""
```

Comment:
Two things: 

1. Does it make sense for the new experiment tracking modules to live in record linkage model helpers? Right now we only have record linkage models, but in the near future I could see incorporating experiment tracking into other kinds of modeling - like table extraction from PDFs. How well does it generalize currently and would it make any sense to just make it a separate module in `pudl.analysis`? 

2. I think we should include some more detailed instructions on how to use these experiment tracking classes - like what is the high level workflow when adding it into a new model. I think the same could be done with instructions on how to use the dataframe embedding/cleaning module. Where do you think a docstring like this would be best placed? Or is it best formatted as a tutorial notebook?

Reply:
See my comment above on organization stuff.

I've tried to put more details in the docstrings, but I do think a tutorial notebook would be good. I think it would be cool to create a couple tutorial notebooks that show how to set experiment tracking up in a notebook, then how to migrate a model into PUDL. We could use these as sort of scaffolding when we work on new models.